src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -283,28 +293,37 @@ If not specified when using the [open_channel](/atomicdex/api/v20-dev/lightning/
-| their_channel_reserve_sats         | boolean | Optional, defaults to value in `coins` file. The minimum balance that the other node has to maintain on their side, at all times. This ensures that if our counterparty broadcasts a revoked state, we can punish them by claiming at least this value on chain. `Default value:` 1% of channel value. `Minimum value:` 1000 sats |
+| commit_upfront_shutdown_pubkey     | boolean | Optional, defaults to `true`. If `true`, we commit to an upfront shutdown_pubkey at channel open.                                                      |
+| counterparty_locktime              | integer | Optional, defaults to `144`. The number of blocks we require our counterparty to wait to claim their money on chainif they broadcast a revoked transaction. We have to be online at least once during this time to punish our counterparty for broadcasting a revoked transaction. We have to account also for the time to broadcast and confirm our transaction, possibly with time in between to [RBF (Replace-By-Fee)](https://bitcoinops.org/en/topics/replace-by-fee/) the spending transaction.  |
+| negotiate_scid_privacy             | integer | Optional, defaults to `false`. If `true`, we attempt to negotiate the `scid_privacy` (referred to as `scid_alias` in the [BOLTs](https://github.com/lightning/bolts)) option for outbound private channels. This provides better privacy by not including our real on-chain channel UTXO in each invoice and requiring that our counterparty only relay HTLCs to us using the channel's SCID alias.  |
+| their_channel_reserve_sats         | boolean | Optional, defaults to `10000` or 1% of channel value. The minimum balance that the other node has to maintain on their side, at all times. This ensures that if our counterparty broadcasts a revoked state, we can punish them by claiming at least this value on chain. |
+
+<DevComment>
+It would be good to better explain (or link to somthing that explains) what it means to "commit to an upfront shutdown_pubkey at channel open"
+</DevComment>
 
 <Note>
 For GUIs and wallet apps, it is recommended to set `announced_channel` to `false`, as the node is not expected to be reliably online. 
 </Note>
 
 ### LightningChannelOptions
 
+<DevComment>
+Needs default values for `their_channel_reserve_sats`.
+</DevComment>
+
 | Parameter                              | Type    | Description                                                                                   |
 |----------------------------------------|---------|-----------------------------------------------------------------------------------------------|
 | their_channel_reserve_sats             | string  | Optional. Amount (in millionths of a satoshi) charged per satoshi for payments forwarded outbound over the channel.                             |
```

Comment:
`their_channel_reserve_sats` is not part of `LightningChannelOptions`, it's only in `LightningChannelConfig`, I see that you already used the default value there.

Reply:
thanks for clarifying, removed item